app/src/main/res/layout/fragment_media_detail.xml

```diff
@@ -190,6 +204,14 @@
                       tools:text="Description of the media goes here. This can potentially be fairly long, and will need to wrap across multiple lines. We hope it looks nice though." />
                 </LinearLayout>
 
+                <TextView
+                  android:id="@+id/show_caption_description_textview"
+                  style="@style/MediaDetailTextLabelGeneric"
+                  android:layout_width="match_parent"
+                  android:layout_height="wrap_content"
+                  android:layout_marginBottom="@dimen/dimen_10"
+                  android:text="Show Caption And Description in more Language" />
```

Comment:
Please make it localizable. Also, change the text to `In all languages`, which is shorter and understandable enough. Thanks!